package.json

```diff
@@ -79,9 +83,11 @@
 		],
 		"settings": {
 			"react": {
-				"pragma": "h"
+				"pragma": "h",
+				"pragmaFrag": "h.Fragment"
 			}
 		},
+		"ignores": ["test/components.js"],
```

Comment:
Why do we need to ignore it?

Reply:
I couldn't get xo to pass like I mentioned, but I wanted to make sure the tests were passing. When I get xo working I'll put it back 😄